### ISSUE TITLE
Crash on launch

### ISSUE THREAD

**Issue by neomoto7** [neogeno/shango-lightning-wallet#52]
Thank you for trying out Shango Beta.   

Please submit your issues with the following information

- OS (Android/iOS)   
iOS
- Version of OS  
12.0.1 (16A405)

- Device Make and Model   
iPhone XS Max 64GB (MT5C2LL/A)

- Shango Version  
1.08.3

- Device ID (Look in Settings-About Shango)  
Unavailable

After download, launching the app results in a black screen followed by a crash.

## Expected Behavior

App should launch

## Current Behavior

App is crashing

## Possible Solution or Best way to handle this problem

N/A

## Steps to Reproduce

Just launched it and it crashed... attaching screenshot of console since no crash log was created
<img width="1601" alt="screen shot 2018-10-16 at 6 53 53 pm" src="https://user-images.githubusercontent.com/38022358/47052111-0f210080-d175-11e8-9737-81810ac18e80.png">

**Comment by neogeno** [comment]
thanks @neomoto7 This is the first time I have seen this issue and I don't have an iPhone XS Max to test unfortunately. I will try to reproduce this on the simulator and see if I can help after that.

**Comment by neogeno** [comment]
@neomoto7  Can you try to delete the app, re-install and try again?

**Comment by neomoto7** [comment]
Will-do

**Comment by neomoto7** [comment]
Same issue... crash on launch.

Attaching some device logging (covers from re-install through the crash)
[shango_crash.txt](https://github.com/neogeno/shango-lightning-wallet/files/2485404/shango_crash.txt)

**Comment by newbold** [comment]
Same issue here on a 64 GB iPhone Xs. Running iOS 12.0 (haven’t upgraded to 12.0.1 yet). Black screen for a brief moment, then crashes to the home screen.

**Comment by neogeno** [comment]
The next version will upgrade all the dependency libraries to the latest versions, let us see if this helps.

**Comment by neomoto7** [comment]
Still quitting on 1.08.6, however, it doesn’t seem like a crash anymore. 

On a fresh run I get the start of some blue particle animation and some audio but it almost immediately quits out of the app and drops me back to the iOS home screen. 

However when bringing up the iOS multitasking menu, Shango is still visible (which I assume means it’s running?)

Selecting it takes me to the app where I get the same loop of starting the animation and audio just to quit again back to the home screen.

**Comment by neogeno** [comment]
Ok we're getting closer to fixing it it seems it's the video animation that is behaving weird. Would it be possible to send me a video so I can see at what point it quits ?

**Comment by neomoto7** [comment]
Sure... attached
[ScreenRecording_10-22-2018 16-15-41.MP4.zip](https://github.com/neogeno/shango-lightning-wallet/files/2503259/ScreenRecording_10-22-2018.16-15-41.MP4.zip)

**Comment by Jonbros01** [comment]
I got same exact issue, iPhone XS MAX 256gb

**Comment by Jonbros01** [comment]
Just updated and still same immediate crash and close.  Issue not fixed unfortunately.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by neomoto7** [comment]
Yes unfortunately I’m seeing the same. Deleted from my device, re-installed through TestFlight, launched the app, and seeing the same behavior (crash after a split-second of playing the video.)

**Comment by neogeno** [comment]
@marco-palladium  Can you send me a screenshot /video at which point it crashes. So did you get pas the initial video animation?

**Comment by Jonbros01** [comment]
No the animation start for a fraction of a second and the app crash… too fast to capture anything

**Comment by marco-palladium** [comment]
I tried to uninstall the app and reinstall it but there is always the same error. how can it be solved?Inviato dal mio dispositivo Samsung

-------- Messaggio originale --------
Da: Jonbros01 <notifications@github.com> 
Data: 02/11/18  22:12  (GMT+01:00) 
A: neogeno/shango-lightning-wallet <shango-lightning-wallet@noreply.github.com> 
Cc: marco-palladium <casa.pallini@alice.it>, Mention <mention@noreply.github.com> 
Oggetto: Re: [neogeno/shango-lightning-wallet] Crash on launch (#52) 

No the animation start for a fraction of a second and the app crash… too fast to capture anything

**Comment by neomoto7** [comment]
So I just downloaded the 1.08.6 (3) build and it looks like the behavior changed slightly. Prior to installing the update I deleted the former build from my device. 

The first time I ran the update I was able to see the Shango logo and it looked like the UI was visible for a split second before it crashed back to the home screen. 

On subsequent re-runs it would simply flash the Shango logo image briefly then immediately quit. 

I recorded video of the process which can be viewed here: https://drive.google.com/file/d/1iEbX_Hf9CRPMN2eD839wYZWz6P69nlz7/view?usp=drivesdk

**Comment by marco-palladium** [comment]
I have the same problem as neomoto7, I sincerely hope they can solve

**Comment by Jonbros01** [comment]
Its fixed!  I can now use the app

**Comment by marco-palladium** [comment]
in practice what should I do? excuse my ignorance

**Comment by marco-palladium** [comment]
in practice what should I do? excuse my ignorance



  ----Messaggio originale----
 
 Da: notifications@github.com
 
 Data: 20-nov-2018 0.24
 
 A: "neogeno/shango-lightning-wallet"<shango-lightning-wallet@noreply.github.com>
 
 Cc: "marco-palladium"<casa.pallini@alice.it>, "Mention"<mention@noreply.github.com>
 
 Ogg: Re: [neogeno/shango-lightning-wallet] Crash on launch (#52)
 

 

 
@neomoto7@marco-palladium@Jonbros01 Can you all please check the latest build number 7 - it seems one of the libraries used had a known problem with A8 CPUs
